### ISSUE TITLE
need to clarify: should implementation ignore conflicting information in the input object?

### ISSUE THREAD

**Issue by mr-c** [common-workflow-language/common-workflow-language#808]
Conformance test 1 include an input object where one of the inputs, a `File`, has an incorrect `size` and `checksum`.

There appears to be no guidance in CWL v1.0 about this situation.

I feel that ignoring this is bad; what if the `File` was a remote resource and the user had specified the `size` and `checksum` as an indication of what they expected.

- [ ] Clarify the intent
- [ ] Document in v1.1
- [ ] Adjust the conformance tests to implement this in a specific test or tests

**Comment by tetron** [comment]
I agree, it should check and raise an error if it does not match.

The alternative would be to ignore it and overwrite the fields, which I think is what cwltool does currently, but I think enforcing consistency would be more valuable.

**Comment by bogdang989** [comment]
Yes, I agree. As specifying these fields is not needed for execution, if user explicitly provides them they could be used for validating.
